src/pan3d/utils/common.py

```diff
@@ -83,6 +84,17 @@ def __init__(self, xarray=None, source=None, server=None, local_rendering=None):
 
         self.ui = None
 
+        # Initialize source
+        self.xarray = None
+        self.source = vtkXArrayRectilinearSource()
```

Comment:
should be None

Reply:
should we remove `self.xarray` altoghther?
i.e. make the explorer only rely on the source (vtkAlgorithm)

Reply:
no, having the xarray is useful